plugins/minecraft/minecraft-users-ram_

```diff
@@ -1,16 +1,20 @@
 #!/bin/bash
 # I'm not the author, just wanted to add this plugin to github
 #  Author & instructions: http://wiki.natenom.name/minecraft/munin-plugin
+#  CPU usage update and translation by Leandro Späth (https://github.com/LeoDJ)
 
 LC_LANG=C
 MC_PORT=${0##*_}
  
 if [ "$1" = "config" ]
 then
+    printf 'graph_category minecraft\n'
+    printf 'graph_scale no\n'
```

Comment:
Please use echo here - printf is not required and just adds visual noise due to the linebreaks.